src/ui_fsmenu/addons.cc

```diff
@@ -463,10 +463,10 @@ AddOnsCtrl::AddOnsCtrl(MainMenu& fsmm, UI::UniqueWindow::Registry& reg)
 			text += '\n';
 			for (const auto& pair : upgrades) {
 				if (pair.second) {
-					text +=
-					   (boost::format(_("\n· %1$s (%2$s) by %3$s")) % pair.first.descname() %
-					    (pair.first.verified ? _("verified") : _("NOT VERIFIED")) % pair.first.author())
-					      .str();
+					text += (boost::format(_("\n· %1$s (%2$s) by %3$s")) % pair.first->descname() %
+					         (pair.first->verified ? _("verified") : _("NOT VERIFIED")) %
+					         pair.first->author())
+					           .str();
```

Comment:
Are sure none of such texts is cached in some GUI or such?

Reply:
This and similar code pieces are used to assemble the message for a confirmation dialogue. The only place where an add-on's strings are shown for a longer time are the Installed and Browse tabs, and these are fully updated (via `rebuild()`) after every change.